addons/web_editor/static/src/js/editor/odoo-editor/src/utils/sanitize.js

```diff
@@ -136,7 +136,11 @@ function sanitizeNode(node, root) {
         !(
             node.attributes?.length === 1 &&
             node.hasAttribute('data-oe-zws-empty-inline') &&
-            (node.textContent === '\u200B' || node.previousSibling.textContent === '\u200B'))
+            (node.textContent === '\u200B' || node.previousSibling.textContent === '\u200B')) &&
+        // In some cases, we want to prevent merging identical elements (E.g.
+        // adjacent `.o_text_highlight_item` wrappers for text highlight lines).
+        ![...node.childNodes].filter(node => node.classList
+            && node.classList.contains("o_content_no_merge")).length
```

Comment:
If I understood correctly you want to check that none of the children have the class `o_content_no_merge` so it's better to do this I think.
```suggestion
        ![...node.childNodes].some(childNode => childNode.classList?.contains("o_content_no_merge"))
```

Reply:
Already updated :wink: